pubsublite/admin.go

```diff
@@ -147,9 +147,41 @@ func (ac *AdminClient) Topics(ctx context.Context, parent string) *TopicIterator
 	}
 }
 
+// OffsetLocation refers to the location of an offset with respect to the
+// message backlog.
+type OffsetLocation int
+
+const (
+	// UnspecifiedOffsetLocation represents an OffsetLocation that has not
+	// been set.
+	UnspecifiedOffsetLocation OffsetLocation = iota
+
```

Comment:
Let's omit this for now to disallow its usage. We can add it later if there's a need for an "unspecified" flag somewhere. End can be initialized with `iota + 1`.

Reply:
Done.